omod/src/test/java/org/openmrs/module/reportingrest/web/resource/ReportRequestResourceTest.java

```diff
@@ -60,6 +72,140 @@ public void setUp() throws Exception {
 	private ReportRequestResource getResource() {
 		return (ReportRequestResource) Context.getService(RestService.class).getResourceBySupportedClass(ReportRequest.class);
 	}
+
+	@Test
+	public void testCreateScheduledReport() throws Exception {
+		String reportDefinitionJson = "{"
+				+ "\"parameterizable\":{"
+				+ "   \"uuid\":\"" + REPORT_DEFINITION_UUID + "\""
+				+ "},"
+				+ "\"parameterMappings\":{"
+				+ "   \"endDate\":\"2017-01-31\","
+				+ "   \"startDate\":\"2017-01-01\"}"
+				+ "}";
+
+		String reportRequestJson = "{\n" +
+				"  \"status\": \"SCHEDULED\",\n" +
+				"  \"priority\": \"HIGHEST\",\n" +
+				"  \"reportDefinition\":" + reportDefinitionJson + "," +
+				"  \"renderingMode\": \"org.openmrs.module.reporting.report.renderer.CsvReportRenderer\",\n" +
+				"  \"schedule\": \"0 42 15 8 2 ? 2018\"\n" +
+				"}";
+
+		SimpleObject properties = SimpleObject.parseJson(reportRequestJson);
+		RequestContext context = new RequestContext();
+		context.setRepresentation(Representation.DEFAULT);
+		SimpleObject response = (SimpleObject) getResource().create(properties, context);
+		assertNotNull(response.get("uuid"));
+		assertEquals(response.get("status"), ReportRequest.Status.SCHEDULED);
+		assertEquals(response.get("priority"), ReportRequest.Priority.HIGHEST);
+		SimpleObject resultObject = (SimpleObject) response.get("renderingMode");
+		String rendererType = (String) resultObject.get("rendererType");
+		assertEquals(rendererType, "org.openmrs.module.reporting.report.renderer.CsvReportRenderer");
+
+		ReportRequest request = getResource().getByUniqueId((String) response.get("uuid"));
+		assertThat(request.getReportDefinition().getParameterizable().getUuid(), is(REPORT_DEFINITION_UUID));
```

Comment:
per my global comment, this test should not need to assert on the parameter mappings, because that's handled by the testCreateWithReportDefinitionAsParameters test.